src/neptune/internal/utils/logger.py

```diff
@@ -39,10 +40,17 @@ def stream(self):
         return sys.stdout
 
 
-logger = logging.getLogger(LOGGER_NAME)
+def get_logger():
+    return logging.getLogger(NEPTUNE_LOGGER_NAME)
 
-logger.propagate = False
-logger.setLevel(level=logging.DEBUG)
-stdout_handler = GrabbableStdoutHandler()
-stdout_handler.setFormatter(logging.Formatter("%(message)s"))
-logger.addHandler(stdout_handler)
+
+def set_up_logging():
```

Comment:
According to this: https://docs.python.org/3/library/logging.html#thread-safety the `logging` is thread-safety so the previous code was actually fine and there is no need to initialize it in `__init__`.

Reply:
What would you consider a better alternative for placement of this setup (it has to be done somewhere)?
I can only think of
1. checking if the call to `get_logger` is the first one, and if that is the case, do the setup. But I see no advantages and additionally, we have to add an extra if statement in `get_logger`.
2. executing `set_up_logging` inside `src/neptune/interal/utils/logger.py'

Reply:
-> 2. 
Like before, create a global variable `logger = get_logger()` in `src/neptune/interal/utils/logger.py` and just import it with `from neptune.internal.utils.logger import logger`. It's safe cause it's thread-safe and it will be evaluated with the first import.